products/ocp4/profiles/moderate-node.profile

```diff
@@ -39,6 +39,7 @@ description: |-
 # CM-6 CONFIGURATION SETTINGS
 # CM-6(1) CONFIGURATION SETTINGS | AUTOMATED CENTRAL MANAGEMENT / APPLICATION / VERIFICATION
 extends: cis-node
+extends: moderate-node-rev-4
 
 filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "ocp4-node-on-sdn" in platforms or "ocp4-node-on-ovn" in platforms'
 
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.